src/main/java/org/hmhb/oauth/DefaultGoogleOauthService.java

```diff
@@ -36,6 +42,33 @@ public GoogleTokenResponse getTokenResponse(
                     redirectUri
             ).execute();
 
+            GoogleCredential credential = new GoogleCredential()
+                    .setAccessToken(
+                            oauthResponse.getAccessToken()
+                    );
+
+            Plus plus = new Plus.Builder(
+                    new NetHttpTransport(),
+                    new JacksonFactory(),
+                    new HttpRequestInitializer() {
+                        @Override
+                        public void initialize(HttpRequest request) throws IOException {
+                            // empty
+                        }
+                    }
```

Comment:
Some of them have the, others don't (or maybe it depends on library versions or something).